components/GameScreen.vue

```diff
@@ -1,10 +1,14 @@
 <script setup lang="ts">
 import { Application, Graphics, Text, FillGradient, Assets, type Texture, Sprite } from "pixi.js";
+import { getSmoothZoomScreen } from "~/other/zoomUtils";
 
+const { data: user } = await useFetch("/api/auth/user");
 const refreshIntervalMs = 1000;
-const zoomSpeed = 0.05;
+const zoomSpeed = 0.1;
 const minZoom = 1;
 const maxZoom = 3;
+const isFollowing = ref<boolean>(false);
+const zoomInToPlayerFn = ref<(() => void) | null>(null);
```

Comment:
Let's either create 2 different functions `zoomIn` and `resetZoom` or update the function name to say that it can do one or the other.

Ideally, we get rid of this function ref, but we can do this in a follow-up.

```suggestion
const zoomInToPlayerOrRestZoomFn = ref<(() => void) | null>(null);
```